### ISSUE TITLE
model links data.dmlc.ml are broken

### ISSUE THREAD

**Issue by jnorwood** [apache/incubator-mxnet#14507]
The model links referencing data.dmlc.ml in the model zoo and inference examples are broken.
To fix this, it appears there are more than one issue.  First, there are no longer gzipp'd files, and second the data.dmlc.ml site is no longer the correct source for wget.

so, this works:

`wget http://data.mxnet.io/models/imagenet/resnet/18-layers/resnet-18-symbol.json`

and this works:

`wget http://data.mxnet.io/models/imagenet/resnet/18-layers/resnet-18-0000.params`

and this works:

`wget https://github.com/dmlc/web-data/raw/master/mxnet/neural-style/model/vgg19.params`

but the links on [https://mxnet.apache.org/model_zoo/index.html](url)

for models and param files are of the form 
`http://data.dmlc.ml/models/imagenet/squeezenet/squeezenet_v1.1-symbol.json`

and the tgz download doc in [https://github.com/dmlc/mxnet-model-gallery/blob/master/imagenet-1k-vgg.md](url)  uses
`wget http://data.dmlc.ml/mxnet/models/imagenet/vgg/vgg16.tar.gz`

and that is true for the other models under [https://github.com/dmlc/mxnet-model-gallery](url)

So, basically, just look through all the docs and examples and change data.dmlc.ml to whatever is supposed to be the correct address for the downloads.  It isn't evident to me if it should be coming from the github raw link above or the data.mxnet.io links, but data.dmlc.ml is simply broken.

**Comment by zachgk** [comment]
According to https://lists.apache.org/thread.html/129d8cb92a978d59ac7dd038c86ffb2db82dc708a3bc09da23727072@%3Cdev.mxnet.apache.org%3E, the change should be `data.dmlc.ml => data.mxnet.io`

@mxnet-label-bot add [Doc, Example, Good first Issue, Call for Contribution]

**Comment by lanking520** [comment]
My pr also crashed by that: https://github.com/apache/incubator-mxnet/pull/14474

**Comment by jnorwood** [comment]
Let me add that the mxnet/cpp-package/example/inference/unit_test_inception_inference.sh is also has the line
`wget -nc http://data.dmlc.ml/mxnet/models/imagenet/inception-bn.tar.gz`
so I presume it is also broken.
That probably needs to be separated to three commands, one for .json and one for .params files, and one for the synset.txt.   The  data.dmlc.ml needs to change to data.mxnet.io

**Comment by jnorwood** [comment]
i got that unit_test_inception_inference.sh test to work with the following changes
`wget -nc -O model/Inception-BN-symbol.json http://data.mxnet.io/mxnet/models/imagenet/inception-bn/Inception-BN-symbol.json
wget -nc -O model/synset.txt http://data.mxnet.io/mxnet/models/imagenet/synset.txt
wget -nc -O model/Inception-BN-0126.params http://data.mxnet.io/mxnet/models/imagenet/inception-bn/Inception-BN-0126.params?raw=true
`
plus ,remove the tar line

**Comment by lebeg** [comment]
I'm working on the fixes here: https://github.com/apache/incubator-mxnet/pull/14504

**Comment by lanking520** [comment]
Now fixed are applied. @jnorwood please feel free to verify and close this issue.

**Comment by jnorwood** [comment]
> Now fixed are applied. @jnorwood please feel free to verify and close this issue.
Thanks for fixing those.  I still found references to data.dmlc.ml in the following files.  

docs/model_zoo/index.md

example/rcnn/README.md

tools/coreml/README.md

tools/coreml/test/test_mxnet_models.py

**Comment by jrosebr1** [comment]
The `http://data.dmlc.io/mxnet/models/imagenet/vgg/vgg16.tar.gz` link is returning a 403 error. It looks like you're using AWS but the .tar.gz file permissions are not set for public URLs. Can that URL (and the other model URLs) be updated?

Thank you!

**Comment by lebeg** [comment]
@jrosebr1 the link you provided is a bit confusing, it should be either `http://data.dmlc.ml/...` or `http://data.mxnet.io/...`.

In any way check latest master that has #14504 merged. If you notice some other link failing let us now.

**Comment by jnorwood** [comment]
there are more broken links still referencing data.dmlc.ml. for example:
`grep data.dmlc.ml */*/* 2>/dev/null`
finds several in `docs/model_zoo/index.md` and a few others
and you can repeat this and find several more in deeper subdirectories.
`grep data.dmlc.ml */*/*/* 2>/dev/null`
finds a couple in `tools/coreml/test/test_mxnet_models.py`
That's all I can find that are remaing, related to this issue.

**Comment by jrosebr1** [comment]
@lebeg Sorry for my misunderstanding. What is the correct URL to download the VGGNet (trained on ImageNet) model?

**Comment by jnorwood** [comment]
[http://data.mxnet.io/models/imagenet/vgg/vgg16-0000.params](url)
[http://data.mxnet.io/models/imagenet/vgg/vgg16-symbol.json](url)

These above work,  but the [https://mxnet.apache.org/model_zoo/index.html](url) is still not updated, else I would close this.

**Comment by jrosebr1** [comment]
Thanks @jnorwood!

**Comment by IvyBazan** [comment]
I've updated the remaining http://data.dmlc.ml/ links that I found and verified them in #15065
